Cargo.toml

```diff
@@ -82,13 +82,13 @@ sqlx = { version = "0.8.2", features = [
     "uuid",
 ], default-features = false }
 stdext = "0.3.3"
-tap_aggregator = { version = "0.5.5", default-features = false }
-tap_core = { version = "4.1.3", default-features = false }
-tap_graph = { version = "0.3.3", features = ["v2"] }
+tap_aggregator = { version = "0.5.6", default-features = false }
+tap_core = { version = "4.1.4", default-features = false }
+tap_graph = { version = "0.3.4", features = ["v2"] }
 tempfile = "3.8.0"
 test-log = { version = "0.2.12", default-features = false }
 test-with = "0.14.6"
-thegraph-core = { version = "0.15.0", features = [
+thegraph-core = { version = "0.15.1", features = [
```

Comment:
This gives us the [`CollectionId`](https://docs.rs/thegraph-core/0.15.1/thegraph_core/struct.CollectionId.html) type.